packages/components/src/VirtualizedList/CellDatetime/CellDatetime.component.js

```diff
@@ -13,6 +13,9 @@ import TooltipTrigger from '../../TooltipTrigger';
 const DATE_TIME_FORMAT = 'YYYY-MM-DD HH:mm:ss';
 
 export function computeValue(cellData, columnData, t) {
+	if (!cellData) {
+		return t('NOT_AVAILABLE', { defaultValue: 'Not available' });
```

Comment:
I'm wondering, instead of setting a string in the cell, why not letting the cell empty ?